content/blog/2017/2017-01-02-jenkins-lifx-notifier-plugin.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Jenkins LIFX notifier plugin"
+tags:
+- plugins
+- lifx
+author: vgaidarji
+---
+
+[NOTE]
+====
+This is a link:http://vgaidarji.me/blog/2017/01/02/jenkins-lifx-notifier-plugin/[cross post] by link:https://github.com/vgaidarji[Veaceslav Gaidarji], open source developer and contributor to the Jenkins/Bitrise projects.
+====
+
+Some time ago I've encountered a link:http://www.lifx.com[LIFX] smart bulbs. These are the bulbs with a chip inside - 50% bulb, 50% chip. There're mobile applications for easy configuration and remote control of the bulb. Nothing special here, it simply works and is very convenient to have such bulbs in dormitory.
+
+=== Brilliant idea time
+
+[quote]
+____
+99% of ideas which come to our minds either were already implemented by someone else or they are shit.
```

Comment:
It's mine :) what's wrong with it? should I be more polite here and replace "shit" with more polite version?